eng/Build.props

```diff
@@ -6,4 +6,11 @@
     <ProjectToBuild Include="$(RepoRoot)src/VirtualMonoRepo/InitializeVMR.proj" BuildInParallel="false" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(DotNetBuildPass)' != '' and '$(DotNetBuildPass)' != '1'">
```

Comment:
Is this conditional required? I thought that the build pass attribute is only used when DotNetBUild==true?

Reply:
Unfortunately, yes. Without it, the ProjectToBuild item won't be ignored in non-VMR builds. That's because of these lines: https://github.com/dotnet/arcade/blob/964e434191bd3ca5675743d08ff742ec0f1e79a9/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L79-L82

We could remove that condition in Arcade but I would first like to hear from @jkoritzinsky to better understand the reasoning behind it.